client/log_verifier.go

```diff
@@ -29,16 +29,16 @@ import (
 
 // LogVerifier contains state needed to verify output from Trillian Logs.
 type LogVerifier struct {
-	hasher hashers.LogHasher
-	pubKey crypto.PublicKey
+	Hasher hashers.LogHasher
+	Pubkey crypto.PublicKey
```

Comment:
`PubKey` is probably more [normal](https://golang.org/doc/effective_go.html#mixed-caps)